ECMAScript/react-native/README.md

```diff
@@ -7,23 +7,27 @@
 
 ## Installation
 
-`@quiltt/react-native` expects `react`, `react-native`,`react-native-webview` and `core-js` as peer dependencies.
+`@quiltt/react-native` expects `react`, `react-native`,`react-native-webview`, `base-64` and `react-native-url-polyfill` as peer dependencies.
 
 ```shell
 $ npm install react-native-webview
-$ npm install core-js
+$ npm install base-64
+$ npm install react-native-url-polyfill
```

Comment:
why do people need to install these if they're full dependencies?

Reply:
it's peer dependencies, not included in the library, but the library expect the app has `base-64` and `react-native-url-polyfill` intalled already.

Just like if you want to use `@quiltt/react`, you expect the app has `react` installed.